dataset/dictionary/talent-materials.ts

```diff
@@ -73,6 +73,30 @@ export default [
     pronunciationJa: "しぶんのてつがく",
     tags: [ "talent-material", "mondstadt" ],
   },
+  {
+    en: "Ascended Sample: Knight",
+    ja: "昇揚のサンプル「ナイト」",
+    zhCN: "升扬样本·骑士",
+    zhTW: "升揚樣本·騎士",
+    pronunciationJa: "しょうようのサンプルナイト",
+    tags: [ "talent-material", "mondstadt" ],
```

Comment:
The drop items from weekly bosses are categorized as `drop-boss`.

```suggestion
    tags: [ "drop-boss", "mondstadt" ],
```